### ISSUE TITLE
Create Accessibility Conformance Documentation

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#99671]
GitHubTags:#A11y_VSCode;#A11ySev1;#A11yTCS;#A11yMAS;#Benchmark;#Win10;#Visual Studio Code;#CAITestPass_May2020;#A11yAAT;#MAC
 **Recommendation:** 
"Fix and close other bugs before starting accessibility conformance documentation.  Any bug that will not be fixed and closed within SLA must be declared in the AAT.  The SLA for this bug is also 60 days."
 [Guide](https://microsoft.sharepoint.com/teams/ceacc/SitePages/Conformance%20Documentation.aspx) for creating conformance documentation.
 "When the AAT entry is completed, please resolve the bug with the AAT URL.  Trusted tester will review the AAT and send comments if any.  Once this review is completed, then mark the AAT entry for review in the AAT itself - this will send the entry to CELA for their review.  When CELA publishes the document, the bug will be closed."

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/99670
